tests/test_tissuemask.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+import pytest
+import cv2
+
+from tiatoolbox.wsicore import wsireader
+from tiatoolbox.tools import tissuemask
+
+
+# -------------------------------------------------------------------------------------
+# Tests
+# -------------------------------------------------------------------------------------
+
+
+def test_otsu_masker(_sample_svs):
+    """Test Otsu's thresholding method."""
+    wsi = wsireader.OpenSlideWSIReader(_sample_svs)
+    mpp = 32
+    thumb = wsi.slide_thumbnail(mpp, "mpp")
+    masker = tissuemask.OtsuTissueMasker()
+    mask_a = masker.fit_transform([thumb])[0]
+
+    masker.fit([thumb])
+    mask_b = masker.transform([thumb])[0]
+
+    assert np.array_equal(mask_a, mask_b)
+    assert masker.threshold is not None
+    assert len(np.unique(mask_a)) == 2
+    assert mask_a.shape == thumb.shape[:2]
+
+
+def test_otsu_grescale_masker(_sample_svs):
```

Comment:
*greyscale

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/3375e6dd0c7ec3c9dd0fccb2ac2e5619a8cc9513